src/Integration.Vsix.UnitTests/CFamily/CFamilyEmbeddedSonarWayRulesTests.cs

```diff
@@ -78,19 +81,21 @@ public void Read_Active_Rules()
         [TestMethod]
         public void Read_Rules_Params()
         {
-            rulesMetadataCache.GetRulesConfiguration("cpp").RulesParameters.TryGetValue("S1311", out var parameters);
```

Comment:
S1311 was marked as obsolete in an earlier version of SQ and removed in the latest release: see [CPP-4615](https://sonarsource.atlassian.net/browse/CPP-4615).

I just picked the first rule that had a `xxx_params.json` file.